packages/pwa-kit-mcp/src/tools/create-new-component.js

```diff
@@ -58,7 +60,11 @@ class CreateNewComponentTool {
             location: z
                 .string()
                 .describe('The location of the component to create')
-                .default(process.env.PWA_STOREFRONT_APP_PATH)
+                .default(process.env.PWA_STOREFRONT_APP_PATH),
+            useCase: z
```

Comment:
For the create-component tool we already have a purpose parameter, could we not drive the hook recommendations based on it,

This might be something to consider in the tool modification